pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -806,6 +806,7 @@ type QuotaSpec struct {
 	MaxBytes *uint64 `json:"maxBytes,omitempty"`
 
 	// MaxSize represents the quota in bytes as a string
+	// +optional
```

Comment:
@leseb Shouldn't the schema be updated with this change? Looks like the crds-gen action is passing though.

Reply:
Hum yes it should, that's strange...

Reply:
Actually no, it's normal since we have the `omitempty`, I've read `+optional` was not mandatory but a good practice.